### DISCUSSION TITLE
Is the --ansi option still significantly affecting the performance as documented?

### DISCUSSION THREAD

**Discussion by ericbn** [junegunn/fzf#3043] [General]
Hi!

Version [0.26.0](https://github.com/junegunn/fzf/blob/01ae621/CHANGELOG.md#0260) change log mentions:

> Significant performance improvement in ANSI code processing

but the [performance](https://github.com/junegunn/fzf/blob/01ae621/README.md#performance) topic in the documentation says:

> `--ansi` tells fzf to extract and parse ANSI color codes in the input, and it makes the initial scanning slower. 

Is the performance documentation still accurate, or can we use `--ansi` starting with version 0.26.0 without worrying about the performance hit anymore?

**Comment by tjex** [comment]
I don't feel like there has been a performance hit after enabling `--ansi` in my default ops. 
However, it may need to come down to a benchmarking test, which I have no skills or know how to do...

**Comment by junegunn** [comment]
It depends on a few parameters, but it still can make fzf slower by 20~30%.

![image](https://user-images.githubusercontent.com/700826/201273193-e2c10ca0-027c-424e-a658-52dad59007b1.png)


Is it noticeable? Probably not, unless you're using fzf with millions of input lines. But there's no point in doing unnecessary extra work so it's not on by default.

**Reply by ericbn** [reply]
Thanks a lot for the benchmarks and the explanation. It makes perfect sense this is not on by default.
